build/Targets/VSL.Settings.targets

```diff
@@ -316,7 +316,7 @@
       $(BuildNumberSuffix.Split('.')[1].PadLeft(2,'0'))
     </BuildNumberPart2>
 
-    <NuGetReleaseVersion>$(RoslynSemanticVersion)</NuGetReleaseVersion>
+    <NuGetReleaseVersion>1.1.1</NuGetReleaseVersion>
```

Comment:
This is incorrect for master -- we should stay on `$(RoslynSemanticVersion)`. I will produce a commit to fix this.